### DISCUSSION TITLE
No Finding Founds from Sonatype LifeCycle (SCA - IQ Server)

### DISCUSSION THREAD

**Discussion by tanatipmfec** [DefectDojo/django-DefectDojo#9196] [Q&A] [answered]
Hi All,

We have tried to test import report which are JSON format and CycloneDX format from Sonatype LifeCycle (SCA - IQ Server) to Defect Dojo and we found issues as below details

- JSON format can import but get only new engagement without any findings
-  CycloneDX format got error from null information and we also tried to adding description but still got same errors
![2023-12-20 17 16 13 View Engagement _ DefectDojo a](https://github.com/DefectDojo/django-DefectDojo/assets/66723024/0d157c11-c34c-44de-85e0-d7ce506270f3)
<img width="1335" alt="MicrosoftTeams-image" src="https://github.com/DefectDojo/django-DefectDojo/assets/66723024/aa6f3059-1578-491d-9380-b8805c8150eb">

Not sure that anyone got similar issues like this or has any recommendations for this one?

**Comment by manuel-sommer** [comment]
Hi @tanatipmfec could you please open up an issue with a sample which produces the error. Maybe the parser has to be fixed. Then, we can reproduce this.

**Reply by tanatipmfec** [reply, marked as answer]
Hi @manuel-sommer

We can import scan result from Sonatype LifeCycle to Defect Dojo but they required JSON Format with RAW data type from solution itself not standard JSON format.
